File: Dom_Ioime/Code/Java/Instructor.java
public class Instructor extends User {
    
    Instructor(){
        super();
    }
    Instructor(String first_name, String last_name, String id){
        super(first_name, last_name, id);
    }

    public void print_schedule(){
        System.out.println("Print Schedule for Instructor!");
    }
    public void print_classlist(){
        System.out.println("Print Classlist for Instructor!");
    }
    public void search_course(){
        System.out.println("Search course for Instructor!");
    }
}

File: Dom_Ioime/Code/Python/Instructor.py
from User import User

class Instructor(User):

    def __init__(self, first_name="First Name", last_name="Last Name", id="W00000000"):
        super().__init__(first_name, last_name, id)
        self.schedule = []
        self.classlist = []

    def print_schedule(self):
        print("Print Schedule for Instructor!")
    def print_classlist(self):
        print("Print Classlist for Instructor!")
    def search_course(self):
        print("Search course for Instructor!")